packages/curve-ui-kit/src/features/user-profile/store.ts

```diff
@@ -5,6 +5,8 @@ import { devtools, persist } from 'zustand/middleware'
 import type { PersistOptions } from 'zustand/middleware/persist'
 import type { ThemeKey } from '@ui-kit/themes/basic-theme'
 
+export const SMALL_POOL_VOLUME = 10000
```

Comment:
bit of a nitpick but I think it should be named `SMALL_POOL_TVL` as we use it in TVL comparisons, not volume

Reply:
I'll fix this in https://github.com/curvefi/curve-frontend/pull/849